### PR TITLE
remove easy_install from documentation (issue #276)

### PR DESCRIPTION
**Pull request by sifuraz**
Original issue text:

AFAIK easy_install is deprecated. It confuses new users.
Please remove it from this page:
http://tox.readthedocs.org/en/latest/install.html

### CODE REVIEW

**Comment by obestwalter** [comment]
Hey @sifuraz, thanks. I just did a quick serach though and there are quite a lot of hits for easy_install. Did you check them all, if they apply or need to be kept (for histrical or whatever other reasons)?

https://github.com/tox-dev/tox/search?utf8=%E2%9C%93&q=easy_install&type=

File: doc/example/basic.txt [hunk 1]
```diff
@@ -151,12 +151,7 @@ By default tox uses `pip`_ to install packages, both the
 package-under-test and any dependencies you specify in ``tox.ini``.
 You can fully customize tox's install-command through the
 testenv-specific :confval:`install_command=ARGV` setting.
-For instance, to use ``easy_install`` instead of `pip`_::
-
-    [testenv]
-    install_command = easy_install {opts} {packages}
-
```
Comment (obestwalter) [inline doc/example/basic.txt]: We should not simply delete that but use a different example. I wish we could use conda already, so I am a bit stumped for a good example. pip with different parameters would be something ... or something far out like:

    install_command = sudo apt install {opts} {packages}

which is obviously crazy but shows that you can stick everything in there, whether it makes sense or not.

Comment (sifuraz) [inline doc/example/basic.txt]: @obestwalter Should I go with your example for now then?

Comment (obestwalter) [inline doc/example/basic.txt]: hmm ... no better not - I am good at having really bad ideas. Let's let this sit a bit and see if something else comes up ...

Comment (sifuraz) [inline doc/example/basic.txt]: As for pip with different paramters, there's example just below the removed part.
``` 
    [testenv]
    install_command = pip install --pre --find-links http://packages.example.com --no-index {opts} {packages}```

Comment (obestwalter) [inline doc/example/basic.txt]: ah, o.k. - maybe that is actually enough then ...

**Comment by sifuraz** [comment]
@obestwalter Hello, I see this is still in "changes requested" state, is there something else you would like me to change?

**Comment by obestwalter** [comment]
@sifuraz this is out of scope of this PR, but we really have no proper docs for the installer comand in the actual config docs. So deleting that big chunk gets rid of the only piece of docs for this. I'll have a closer look and merge your PR then, so there is nothing else needed for this particular PR. Thanks!

**Comment by obestwalter** [comment]
Also: the config-v2.txt should not be changed anyway as it's a "historic" document :) - but don't worry about it :)

**Comment by obestwalter** [comment]
Unless you want to fix up your PR yourself, please revert config-v2.txt - cheers.

**Comment by obestwalter** [comment]
flake8 is broken on appveor - has nothing to do with PR.

**Pull request merged**